distage/distage-core/src/test/scala/com/github/pshirshov/izumi/distage/dsl/DSLTest.scala

```diff
@@ -305,15 +306,15 @@ class DSLTest extends WordSpec {
       import BasicCase6._
 
       val definition = new ModuleDef {
-        bindMany[ImplXYZ].to[TraitX].to[TraitY].to[TraitZ]
+        bind[ImplXYZ].to[TraitX].to[TraitY].to[TraitZ]
```

Comment:
Yep! Now to extend the test with other `bind*` variations and it should be ready 👍